libraries/lib-numeric-formats/formatters/ParsedNumericConverterFormatter.cpp

```diff
@@ -1019,9 +1019,10 @@ class ParsedNumericConverterFormatterFactory final :
        : mType { std::move(type) }
        , mFormat { std::move(format) }
    {
-      // Only TIME with # in the format depends on sample rate now
-      mDependsOnSampleRate = mType != NumericConverterType_TIME ||
-                             mFormat.Debug().find(L'#') != wxString::npos;
+      // # in the format means that Sample Rate is used
+      // to calculate the values. Otherwise, Sample Rate is optional for
+      // the TIME and DURATION formats and ignored by the others.
+      mDependsOnSampleRate = mFormat.Debug().find(L'#') != wxString::npos;
```

Comment:
But now this means that frequency and bandwidth converters depended on sample rate before, but now they do not.  Is that correct?

Reply:
Yes I see it is correct, now that I reexamined all uses of `mSampleRate`.  It really is only those formats contianing `#` that have any dependency on it.